app/models/task.rb

```diff
@@ -0,0 +1,20 @@
+# == Schema Information
+#
+# Table name: tasks
+#
+#  id          :uuid             not null, primary key
+#  name        :string
+#  description :text
+#  state       :integer
+#  project_id  :uuid
+#  created_at  :datetime         not null
+#  updated_at  :datetime         not null
+#
+
+class Task < ActiveRecord::Base
+  belongs_to :project
+
+  enum state: %i{todo in-progress done}
+
+  validates :project, presence: true
```

Comment:
Agree, it doesn't make sense to have tasks without name; I just defaulted state to "todo" so the state validation it's not necessary.